<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfessorChangeDialog</class>
 <widget class="QDialog" name="ProfessorChangeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>391</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="grdAvailability">
      <property name="styleSheet">
       <string notr="true">QHeaderView::section {
    background-color: rgb(11, 117, 250);
    height: 32px;
	border-bottom-style: dotted;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnUpdate">
        <property name="styleSheet">
         <string notr="true">QPushButton {
      background-color: rgb(11, 117, 250);
	  border-style: outset;
	  border-width: 2px;
	  border-radius: 15px;
	  border-color: beige;
	  font: bold 14px;
	  color: white;
	  min-width: 10em;
	  padding: 6px;
 }


QPushButton:hover:!pressed
{
      background-color: rgb(3, 7, 128);
	  border-style: outset;
	  border-width: 2px;
	  border-radius: 15px;
	  border-color: beige;
	  font: bold 14px;
	  color: white;
	  min-width: 10em;
	  padding: 6px;
}
</string>
        </property>
        <property name="text">
         <string>Actualizar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExit">
        <property name="styleSheet">
         <string notr="true">QPushButton {
      background-color: rgb(250, 78, 78);
	  border-style: outset;
	  border-width: 2px;
	  border-radius: 15px;
	  border-color: beige;
	  font: bold 14px;
	  color: white;
	  min-width: 10em;
	  padding: 6px;
 }


QPushButton:hover:!pressed
{
      background-color: rgb(204, 0, 0);
	  border-style: outset;
	  border-width: 2px;
	  border-radius: 15px;
	  border-color: beige;
	  font: bold 14px;
	  color: white;
	  min-width: 10em;
	  padding: 6px;
}</string>
        </property>
        <property name="text">
         <string>Salir</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>4</y>
     <width>324</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lblName">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="txtName">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
